doc/_admin-guide/060_Sources/175_syslog-otlp/000_otlp_source_options.md

```diff
@@ -0,0 +1,194 @@
+* ca-file()
+* key-file()
+* cert-file()
+* peer-verify()
+  * required-trusted
+  * required-untrusted
+  * optional-trusted
+  * optional-untrusted
+
+#### Example: configure an syslog-ng-otlp() source using auth(tls())
+
+```config
+destination {
+    syslog-ng-otlp(
+      url("your-otel-server:12346")
+      auth(
+        tls(
+          ca-file("/path/to/ca.pem")
+          key-file("/path/to/key.pem")
+          cert-file("/path/to/cert.pem")
+        )
+      )
+    );
+  };
+```
+
+## chain-hostname()
+
+|   Type:|       `yes`, `no`|
+|Default:|              `no`|
+
+*Description:* This option can be used to enable or disable the chained hostname format. For more information, see the chain-hostnames() global option.
+
+{% include doc/admin-guide/options/channel-args.md %}
+
+## concurrent-requests()
+
+|   Type:|       integer|
+|Default:|             2|
+
+*Description:* This option configures the upper limit of in-flight gRPC requests per worker. It is advisd to set this value in the range of 10s or 100s when there are a high number of clients sending simultaneously. In an optimzed solution, the number of `workers()` and `concurrent-requests()` is greater than or equal to the number of clients. However, this can cause an increase in memory usage.
+
+## default-facility()
+
+|   Type:|       facility string|
+|Default:|                  kern|
+
+*Description:* This option assigns a facility value to messages received from the file source if the message does not specify one.
+
+## default-level()
+
+|   Type:|       string|
+|Default:|       notice|
+
+*Description:* This option defines the default level value if the `PRIORITY` entry does not exist.
+
+## default-priority()
+
+|   Type:|       priority string|
+|Default:|                      |
+
+*Description:* This option assigns an emergency level to messages received from the file source if the message does not specify one, for example, `default-priority(warning)`.
+
+## dns-cache()
+
+|   Type:|       `yes`, `no`|
+|Default:|              `no`|
+
+*Description:* This option enables or disables the DNS cache usage.
+
+## ebpf()
+
+Available in syslog-ng OSE 4.2 and later versions.
+
+If this option is not used, the kernel chooses the receive socket for a specific UDP randomly based on the source IP/port of the sender. It is possible to customize this algorithm using the Extended Berkeley Packet Filter (eBPF) plugin. The `ebpf()` option changes the `SO_REUSEPORT` algorithm of the kernel, to place messages randomly into one of the UDP sockets. The decision which UDP socket buffer a datagram is placed is made for every datagram, and not once for every stream. This results in the percet load-balancing of messages across the set of UDP sockets. While this resolves the imbalance between the sockets and results in perfect load balancing, the order of messages from the same sender is lost, which is the drawback of increased throughput.
+
```

Comment:
en example config would be useful here I guess

Reply:
Added one based on [this](https://github.com/syslog-ng/syslog-ng/pull/4365).
Let me know if it's not quite right.